src/api.d.ts

```diff
@@ -0,0 +1,16 @@
+export interface InjectOptions {
+  /**
+   * @default '__POSTJECT'
+   */
+  machoSegmentName?: string;
+  /**
+   * @default false
+   */
+  overwrite?: boolean;
+  /**
+   * @default "POSTJECT_SENTINEL_fce680ab2cc467b6e072b8b5df1996b2"
+   */
+  sentinelFuse?: string;
+}
+
+export function inject(filename: string, resourceName: string, resourceData: Buffer, options: InjectOptions): Promise<void>;

```

Comment:
```suggestion
export function inject(filename: string, resourceName: string, resourceData: Buffer, options?: InjectOptions): Promise<void>;

```

Was just trying out a copy of this PR locally, it looks like (from the use of optional chaining in the implementation: https://github.com/nodejs/postject/blob/main/src/api.js#L6 ) that `options` is allowed to be undefined.